src/components/FamilyManifestModal/queries.js

```diff
@@ -106,14 +106,14 @@ export const generateFamilyManifestModalProps = async ({ api, sqon }) => {
     participantIds,
     familyMembersWithoutParticipantIds,
   } = await familyMemberAndParticipantIds({ api, sqon });
-  const dataTypes = await dataTypesFromFamilyMemberIds({
-    api,
-    familyMemberIds: familyMembersWithoutParticipantIds,
-  });
   const { participantFilesCount, participantFilesSize } = await participantsFilesCountAndSize({
     api,
     sqon,
   });
+  const dataTypes = await dataTypesFromFamilyMemberIds({
+    api,
+    familyMemberIds: familyMembersWithoutParticipantIds,
+  });
```

Comment:
Should we do a Promise all here since there's no dependency? 
```
const [{ participantFilesCount, participantFilesSize }, dataTypes] = await Promise.all([
  participantsFilesCountAndSize(...),
  dataTypesFromFamilyMemberIds(...),
]);
```

Reply:
`dataTypes` actually relies on `familyMembersWithoutParticipantIds` from `familyMemberAndParticipantIds` :) but definitely makes sense for the first two. Addressed! :)